src/codelabs/implicit-animations.md

```diff
@@ -207,20 +207,20 @@ example and click the **Show details** button to trigger the animation.
 The [Fade-in text effect][] example demonstrates the following features
 of `AnimatedOpacity`:
 
-* `AnimatedOpacity` listens for state changes in its `opacity` property.
-* Whenever `opacity` changes, `AnimatedOpacity` automatically animates the
+- `AnimatedOpacity` listens for state changes in its `opacity` property.
+- Whenever `opacity` changes, `AnimatedOpacity` automatically animates the
   widget's transition to the new value for `opacity`.
-* `AnimatedOpacity` requires a `duration` parameter to define the time it takes
+- `AnimatedOpacity` requires a `duration` parameter to define the time it takes
   to animate the transition between an old `opacity` value and a new one.
 
 {{site.alert.secondary}}
-  Note that implicit animations can only animate properties of a parent
-  `StatefulWidget`, so this example begins with the `FadeInDemo` widget that
+  Implicit animations can only animate properties of a parent
+  `StatefulWidget`. This example begins with the `FadeInDemo` widget that
```

Comment:
This change loses an important connection between the two parts. The first part serves to explain the second part, but that's not clear now.